common/models/catalog.js

```diff
@@ -29,6 +29,10 @@ class Catalog {
     return service;
   }
 
+  getServiceName(id) {
```

Comment:
Is this helper really required? `.name` seems pretty standard. But if you are convinced, can ignore this comment.

Reply:
Kept it make the code reusable :)